packages/ruby/Gemfile

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 # Specify your gem's dependencies in readme-metrics.gemspec
 gemspec
 
-gem "json-schema"
-gem "rack-test"
-gem "rake", "~> 12.0"
-gem "rspec", "~> 3.0"
-gem "standard"
-gem "webmock"
-gem "os"
```

Comment:
No longer using `os` as it was only used to help construct the HAR `creator.comment`. Using `RUBY_PLATFORM` now instead.